docs/sources/flow/tutorials/collecting-prometheus-metrics.md

```diff
@@ -10,16 +10,33 @@ Grafana Agent is a telemetry collector with the primary goal of moving telemetry
 # Prerequisites
 
 * [Docker](https://www.docker.com/products/docker-desktop)
-* Clone the [Agent Repository](https://github.com/grafana/agent) `git clone git@github.com:grafana/agent.git`
 
 # Scraping the Agent
 
-To quickly spin up an example environment, run the following: `CONFIG_FILE=agent.flow docker-compose -f ./assets/docker-compose.yaml up` with the [flow file](../assets/flow_configs/agent.flow). Allow the service to run for a few minutes, then navigate to the [metrics browser](http://localhost:3000/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Mimir%22,%7B%22refId%22:%22A%22,%22instant%22:true,%22range%22:true,%22exemplar%22:true,%22expr%22:%22agent_build_info%7B%7D%22%7D%5D).
+Run the following `curl https://raw.githubusercontent.com/grafana/agent/main/docs/sources/flow/tutorials/assets/runt.sh -O && bash ./runt.sh agent.flow`. 
```

Comment:
Are users going to be comfortable running a script like this?  Just want to make sure in the long run that we're doing what people expect. I think what you had before was more typical, like in this tutorial: https://grafana.com/tutorials/play-with-grafana-mimir/.

Reply:
Also, the current approach creates a bunch of files and folders in the current directory.

Personal preference, but how about adding a `mkdir flow_tutorials && cd flow_tutorials` first?

Reply:
I will add the `mkdir` and `cd.` Cloning and moving to the correct directory feels like a lot of friction to me. Will give it some thought to see if there is a cleaner method.